src/components/refactor/dialogs/load-modification/load-modification-dialog.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+    ACTIVE_POWER,
+    EQUIPMENT_ID,
+    EQUIPMENT_NAME,
+    LOAD_TYPE,
+    REACTIVE_POWER,
+} from 'components/refactor/utils/field-constants';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { modifyLoad } from '../../../../utils/rest-api';
+import { sanitizeString } from '../../../dialogs/dialogUtils';
+import yup from '../../utils/yup-config';
+import ModificationDialog from '../commons/modificationDialog';
+import LoadModificationForm from './load-modification-form';
+
+/**
+ * Dialog to create a load in the network
+ * @param studyUuid the study we are currently working on
+ * @param defaultIdValue the default load id
+ * @param currentNode The node we are currently working on
+ * @param editData the data to edit
+ * @param dialogProps props that are forwarded to the generic ModificationDialog component
+ */
+
+const schema = yup
+    .object()
+    .shape({
+        [EQUIPMENT_ID]: yup.string().required(),
```

Comment:
change to   [EQUIPMENT_ID]: yup.string().nullable().required(),

Reply:
not sure why ? This field doesn't accept null or am I missing something ?

Reply:
well, add some text in name field and valid ==> the ID field well show the error

Reply:
fixed